src/components/view/newsView.ts

```diff
@@ -0,0 +1,218 @@
+import Option from "../Dto/Option";
+import RequestParameters from "../Dto/RequestParameters";
+import NewsResult from "../Dto/NewsResult";
+import SourceResult from "../Dto/SourceResult";
+import Article from "../Dto/Article";

```

Comment:
Use alias import in Dto folder
![image](https://user-images.githubusercontent.com/76782175/179418693-70e6765e-3e3d-4d58-bafa-cb84b1bd7906.png)